src/LOAuth.ts

```diff
@@ -57,8 +101,11 @@ class LOAuth {
           window.location.href,
           options
         );
+
+        this._decodeAppState();
```

Comment:
To me it seems like this call here only because the _getAppUri call just below it is expecting to have 'appState' populated - is that right?
If that's the case I think it would be nice to have it in the _getAppUri function instead so it's clearer where it is being used.  And maybe even remove it from the class altogether, just make it a function that returns an object?  You are already calling and using the result in the constructor so it's not jumping out at me why it needs to be persisted

Reply:
Yep- I dig the idea of calling _decodeAppState from within _getAppUri.  Saving/exposing "appState" isn't required for functionality purposes, but is intended to be used by PHC custom apps.  We may change how those params are passed (via path, etc.), and consumers will be able to continue using "auth.appState.account" for example.